tests/testthat/test-githubbio-packages.R

```diff
@@ -0,0 +1,54 @@
+# Run this test for users to make sure the long running job feature
+# of doAzureParallel are still working
+context("long running job scenario test")
+test_that("Long Running Job Test", {
+  testthat::skip_on_travis()
+  credentialsFileName <- "credentials.json"
+  clusterFileName <- "cluster.json"
+
+  doAzureParallel::generateCredentialsConfig(credentialsFileName)
+  doAzureParallel::generateClusterConfig(clusterFileName)
+
+  # set your credentials
+  doAzureParallel::setCredentials(credentialsFileName)
+  cluster <- doAzureParallel::makeCluster(clusterFileName)
+  doAzureParallel::registerDoAzureParallel(cluster)
+
+  opt <- list(wait = FALSE)
+  '%dopar%' <- foreach::'%dopar%'
+  github <- c('azure/rAzureBatch', 'azure/doAzureParallel')
+  res <-
+    foreach::foreach(
+      i = 1:4,
+      .packages = c('httr'),
+      #github = github,
+      .options.azure = opt
+    ) %dopar% {
+      mean(1:3)
+    }
+
+  job <- getJob(res)
+
+  # get active/running job list
+  filter <- filter <- list()
```

Comment:
You can remove the long running job part of this test since you are only testing if the github packages are being installed

Reply:
removed.